astro/src/content/docs/apis/_import-users-request-body.mdx

```diff
@@ -174,6 +174,9 @@ You must provide either the **email** or the **username** field for each User. T
   </APIField>
   <APIField name="users[x].registrations[x].roles" type="Array<String>" optional>
     The list of roles that the User has for this registration.
+    <Aside type="note">
```

Comment:
Let's not use an aside, as I don't see any other place where there's an aside in an api field definition. (I spot checked a few api docs, maybe I missed something).

Reply:
I wasn't going to but then I found this, happy to remove them though. How should we call out these details better so that it grabs someones attention?

https://github.com/FusionAuth/fusionauth-site/blob/7716d90ea5ca0ab8171eac1075436c4a0e10454c/astro/src/content/docs/apis/_import-users-request-body.mdx#L58